src/Parser.js

```diff
@@ -78,22 +80,28 @@ export function parse (value, points, declarations, rules, rulesets) {
 
 				index = length = offset = 0, ampersand = 1, type = temporary = ''
 				break
+			// -
+			case 45:
+				switch (previous) {
+					case 45:
+						variable = 0
+				}
 			default:
 				switch (temporary += from(character), character) {
```

Comment:
why not multiply the character variable once? It is a number.

Reply:
we want to match `:`, it's important for setting `length`

Reply:
we could move it to the the top switch.